framework/wazuh/core/exception.py

```diff
@@ -500,6 +500,15 @@ class WazuhException(Exception):
         3038: "Error while processing extra-valid files",
         3039: "Timeout while waiting to receive a file",
         3040: "Error while waiting to receive a file",
+        3041: "Server status check timed out after adding new servers",
```

Comment:
We could add a comment like the one below for `RBAC` to describe that this set of codes is for `HAProxy Helper` exceptions.